What employees really think about PostHog (#8819)
PostHog/posthog.com: contents/blog/company-survey-h1-2024.md

---
date: 2024-06-25

> title: What employees really think about PostHog

Reviewer: ```suggestion
title: What our team really thinks about PostHog
```

Reviewer: @jtemperton this feels less clunky

author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/survey_hog_bafed58598.jpg
tags:
  - Culture
  - Inside PostHog
---

We want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re publishing them here for anyone to see. This page will always feature the results of our more recent survey.

## How we run our company survey

* We ask 14 questions covering categories such as company, confidence, culture, and growth. 12 of the questions we ask require a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree. These questions also have an optional text field for people to add comments. These questions are listed in the [Benchmarks](#benchmarks) section below. 

We also ask two questions which are mandatory and have open text fields:

* What motivates you right now?
* If you were to leave PostHog, what would the reason be?

We run this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). 

> Our use to run surveys quarterly, but we do them every six months.

Reviewer: ```suggestion
Our use to run surveys quarterly, but we do them every six months.

```

> We run this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). 
> 
> Our use to run surveys quarterly, but we do them every six months.
> ## H1 2024 survey results

Reviewer: ```suggestion
We run this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). We used to run surveys quarterly, but we now do them every six months.

## H1 2024 survey results
```

We had a 79% response rate, or 33 out of 42 eligible respondents, for our H1 2024 survey. For our last survey, which covered Q4 2023, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.

> 

Reviewer: ```suggestion
```

### Highest scores

* **97% positive:** I understand PostHog’s goals and can see how my work contributes to them.
* **97% positive:** PostHog is in a position to really succeed over the next three years.
* **97% positive:** My manager and team around me genuinely care about my wellbeing.

![PostHog company survey](https://res.cloudinary.com/dmukukwp6/image/upload/company_survey_1_036769c290.png)

### Lowest scores

* **82% positive:** The support I am receiving and processes we have in place allow me to do my best possible work.
* **79% positive:** I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies. 
* **79% positive:** I see myself still working at PostHog in two years' time.

> * 

Reviewer: ```suggestion

```

#### Positive:

* Bouncing wacky ideas off of peeps, then building them at warp speed
* Everyone's motivation to grow and see the company succeed.
* Shipping a product people love and use
* I feel bad for people who don't get to work somewhere like PostHog
* I would say PostHog is a place where people who can work independently and without much process and support do best, but for me this 'light-touch' process works
* I really love the weekly context bomb at the all hands

![PostHog company survey](https://res.cloudinary.com/dmukukwp6/image/upload/company_survey_2_b73232d570.png)

#### Constructive:

* I think the lack of general process and over-engineered ownership that is typically a big plus has gotten in the way of some products actually making the strides forward that they could have.
* For the company as a whole, I think things are pretty good, but I do think that satisfaction is heavily dependent on the team you're on.
* Our setup means we are always on the edge of exhaustion because we purposefully keep things lean and tight on numbers.
* All ingredients are there but growing pains are apparent specifically around team management.

> 

Reviewer: ```suggestion
```

---
date: 2024-06-25
title: What employees really think about PostHog
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/survey_hog_bafed58598.jpg
featuredImageType: full
category: Inside PostHog
tags:
  - Culture
---

> At PostHog, we want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re going to publish them here for anyone to see. This page will always feature the results of our more recent survey.

Reviewer: ```suggestion
We want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re publishing them here for anyone to see. This page will always feature the results of our more recent survey.
```

> 

Reviewer: ```suggestion

## How we run our company survey

* We ask 14 questions covering categories such as company, confidence, culture, and growth. 12 of the questions we ask require a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree. These questions also have an optional text field for people to add comments. These questions are listed in the [Benchmarks](#benchmarks) section below. 

We also ask two questions which are mandatory and have open text fields:

* What motivates you right now?
* If you were to leave PostHog, what would the reason be?

We run this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). 

Our use to run surveys quarterly, but we do them every six months.
```

> ### H1 2024 PostHog company survey results

Reviewer: ```suggestion
## H1 2024 survey results
```

We had a 79% response rate, or 33 out of 42 eligible respondents, for our H1 2024 survey. For our last survey, which covered Q4 2023, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.

> We asked 16 questions that required a mix of qualatative and quantative responses. For transparency, we've included details on how we run the survey below. Here's what our team members said.

Reviewer: ```suggestion
```

### Highest scores

* **97% positive:** I understand PostHog’s goals and can see how my work contributes to them.
* **97% positive:** PostHog is in a position to really succeed over the next three years.
* **97% positive:** My manager and team around me genuinely care about my wellbeing.

![PostHog company survey](https://res.cloudinary.com/dmukukwp6/image/upload/company_survey_1_036769c290.png)

### Lowest scores

* **82% positive:** The support I am receiving and processes we have in place allow me to do my best possible work.
* **79% positive:** I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies. 
* **79% positive:** I see myself still working at PostHog in two years' time.
* 
We don't use AI to summarize these results at our scale as we think it's worth a human reading and synthesizing this information properly.

> ## Benchmarks

Reviewer: ```suggestion
### Benchmarks
```

#### Positive:

* Bouncing wacky ideas off of peeps, then building them at warp speed
* Everyone's motivation to grow and see the company succeed.
* Shipping a product people love and use
* I feel bad for people who don't get to work somewhere like PostHog
* I would say PostHog is a place where people who can work independently and without much process and support do best, but for me this 'light-touch' process works
* I really love the weekly context bomb at the all hands

![PostHog company survey](https://res.cloudinary.com/dmukukwp6/image/upload/company_survey_2_b73232d570.png)

#### Constructive:

* I think the lack of general process and over-engineered ownership that is typically a big plus has gotten in the way of some products actually making the strides forward that they could have.
* For the company as a whole, I think things are pretty good, but I do think that satisfaction is heavily dependent on the team you're on.
* Our setup means we are always on the edge of exhaustion because we purposefully keep things lean and tight on numbers.

> * All ingredients are there but growing pains are apparent specifically around team management

Reviewer: ```suggestion
* All ingredients are there but growing pains are apparent specifically around team management.
```

> * ### How we run our company survey
> 
> * We asked 14 questions covering categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree. These questions also had an optional text field for people to add comments. These questions are listed above in the Benchmarks section. 
> 
> We also asked two questions which were mandatory and had open text fields:
> 
> * What motivates you right now?
> * If you were to leave PostHog, what would the reason be?
> 
> We ran this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.

Reviewer: ```suggestion
```

![PostHog company survey](https://res.cloudinary.com/dmukukwp6/image/upload/company_survey_4_f0bec1d01e.png)

### What we’ll do next

Our executive team reads all the responses in full and has already gone through all the results. [Charles](https://posthog.com/community/profiles/28625) is responsible for making sure we identify any actions that might be needed. Where action is needed, it is usually the relevant exec team member who follows up.

> That said, not every comment needs a follow up. The priority here is to collect information and get a sense of overall trends. The survey becomes a failure if it is used as the primary way for people to share feedback with PostHog, something which we encourage them to do through more transparent channels. This is partly why we only do it every six months.

Reviewer: ```suggestion
That said, not every comment needs a follow up. The priority here is to collect information and get a sense of overall trends. The survey becomes a failure if it is used as the primary way for people to share feedback with PostHog, something which we encourage people to do through more transparent channels. This is partly why we only do it every six months.
```

> ---
> date: 2024-06-25
> title: What employees really think about PostHog
> rootPage: /blog
> sidebar: Blog
> showTitle: true
> hideAnchor: true
> author:
>   - james-temperton
> featuredImage: >-
>   https://res.cloudinary.com/dmukukwp6/image/upload/survey_hog_bafed58598.jpg
> featuredImageType: full
> category: Inside PostHog
> tags:
>   - Culture
> ---

Reviewer: ```suggestion
---
date: 2024-06-25
title: What employees really think about PostHog
author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/survey_hog_bafed58598.jpg
tags:
  - Culture
  - Inside PostHog
---
```

---
date: 2024-06-25

> title: What employees really think about PostHog

Reviewer 2: Semantics maybe, but we always say 'team member' rather than 'employee'

Author: i like semantics but in this instance are semantics getting in the way of a good headline? 'team members' doesn't have the same ring as 'employees' for me

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/mykonoshackathon_979cf7bec6.jpeg
featuredImageType: full
category: Inside PostHog
tags:
  - Culture
---

At PostHog, we want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re going to publish them here for anyone to see.

> We had a 79% response rate, or 33 out of 42 eligible respondents. For our last survey, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.

Author: ```suggestion
We had a 79% response rate, or 33 out of 42 eligible respondents, for our H1 2024 survey. For our last survey, which covered Q4 2023, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.
```

> We asked 14 questions based on the ones used by [Culture Amp](https://www.cultureamp.com/). Questions cover categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree:

Reviewer 2: This is a bit out of date now, the questions themselves are basically from a variety of sources, tweaked over time to give us the most useful information so it's basically a custom mix

> We asked 14 questions based on the ones used by [Culture Amp](https://www.cultureamp.com/). Questions cover categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree:

Author: ```suggestion
We asked 14 questions covering categories such as company, confidence, culture, and growth. 12 of the questions we asked required a response ranging from 1, meaning strongly disagree, to 5, meaning strongly agree. These questions also had an optional text field for people to add comments:
```

* PostHog is in a position to really succeed over the next three years.
* I understand PostHog's goals and can see how my work contributes to them.
* At PostHog, we have open and honest two-way communication.
* I receive appropriate recognition for my work at PostHog.
* I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies.
* My manager and team around me genuinely care about my wellbeing.
* The leaders at PostHog keep people informed about what is happening.
* I feel like I am learning and growing at PostHog.
* Generally, I believe my workload is reasonable for my role and I am able to arrange time out from work when I need it.
* The support I am receiving and processes we have in place allow me to do my best possible work.
* I would recommend PostHog as a great place to work.
* I see myself still working at PostHog in two years' time.

> 

Reviewer 2: For each of the above, there is also an optional free text field for people to add comments

> And two of which were open text fields:

Reviewer 2: - These questions are mandatory
- They are particularly useful and often the most acted-upon

> And two of which were open text fields:

Author: ```suggestion
And two of which were mandatory and had open text fields:
```

* What motivates you right now?
* If you were to leave PostHog, what would the reason be?

We took the scores we received and compared them to Culture Amp’s benchmark for ‘new tech’ companies with less than 200 people. We then took the average score out of 5 and multiplied it by 20 to get a percentage. It’s a bit rough, but is useful for comparing our scores against the industry benchmark.

> We ran this survey using the Pulse Surveys app in Slack. Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.

Reviewer 2: We use [Deel Pulse Surveys](https://www.deel.com/plugins/pulse-surveys/)

(Btw once this is done, we can share with the Deel team as they may reshare it)

> We ran this survey using the Pulse Surveys app in Slack. Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.

Author: ```suggestion
We ran this survey using the [Deel Pulse Surveys app in Slack](https://www.deel.com/plugins/pulse-surveys/). Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.
```

> ### Highest scores

Reviewer 2: Displaying these scores against the benchmarks is really powerful here

Author: sorry, you've lost me... what's the benchmark?

Reviewer 2: [See benchmarks here (2/3 down the summary)](https://github.com/PostHog/company-internal/issues/1390)

* **97% positive:** I understand PostHog’s goals and can see how my work contributes to them.
* **97% positive:** PostHog is in a position to really succeed over the next three years.
* **97% positive:** My manager and team around me genuinely care about my wellbeing. 

### Lowest scores

* **82% positive:** The support I am receiving and processes we have in place allow me to do my best possible work.
* **79% positive:** I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies. 
* **79% positive:** I see myself still working at PostHog in two years' time. 

> 

Author: ```suggestion
We don't use AI to summarize these results at our scale as we think it's worth a human reading and synthesizing this information properly.
```

> ### Qualitative questions

Reviewer 2: 'Fun' fact - we don't use AI to summarize this at our scale, we think it's worth a human reading and synthesizing it properly

* **Career growth.** People used phrases such as: start my own company, getting back into management, a better opportunity for my career, a career growth move, and starting my own startup.

* **The work.** People used phrases such as: not feeling like I'm making an impact, getting overwhelmed with non-fun work, stuck too long on challenging, repeating things, and burnout.

* **Company stops growing.** People used phrases such as: business growth stalling, growth stalling or the company failing, and the company isn't growing anymore.

### Highlights from comments

These specific but anonymous quotes are broadly reflective of responses from multiple people. We’ve split them into two categories, positive and constructive, as they cover a wide range of topics.

Positive:

* Bouncing wacky ideas off of peeps, then building them at warp speed
* Everyone's motivation to grow and see the company succeed.
* Shipping a product people love and use
* I feel bad for people who don't get to work somewhere like PostHog
* I would say PostHog is a place where people who can work independently and without much process and support do best, but for me this 'light-touch' process works

> * All-hands is great!

Reviewer 2: You can probably just have the second all hands point only, as they make the same point

> * All-hands is great!

Author: ```suggestion
```

* I really love the weekly context bomb at the all hands

Constructive:

* I think the lack of general process and over-engineered ownership that is typically a big plus has gotten in the way of some products actually making the strides forward that they could have.
* For the company as a whole, I think things are pretty good, but I do think that satisfaction is heavily dependent on the team you're on.
* Our setup means we are always on the edge of exhaustion because we purposefully keep things lean and tight on numbers.
* All ingredients are there but growing pains are apparent specifically around team management

### What we’ll do next

> Our exec team has already gone through all the results and we’ve decided on a bunch of actions that are already in progress with a plan and owner to address them. We’ll run the next survey at the start of Q4 2024 using the same format.

Reviewer 2: We can flesh out more:
- Our executive team reads all responses in full
- Charles is responsible for making sure that we identify an actions that might be needed - typically the follow up occurs over Slack or in a 1-1. A good way to follow up is to ask directly 'in the latest team survey you said X, could you tell me more?' Usually it is the relevant exec team member who follows up here. 
- It is worth noting that not every comment needs a follow up - the priority here is to collect information and get a sense of overall trends. The survey becomes a failure if it is used as the primary way for people to share feedback with PostHog (which we encourage through more transparent channels). This is partly why we only do it every 6 months.

> Our exec team has already gone through all the results and we’ve decided on a bunch of actions that are already in progress with a plan and owner to address them. We’ll run the next survey at the start of Q4 2024 using the same format.

Author: ```suggestion
Our executive team reads all the responses in full and has already gone through all the results. [Charles](https://posthog.com/community/profiles/28625) is responsible for making sure we identify any actions that might be needed. Where action is needed, it is usually the relevant exec team member who follows up.

That said, not every comment needs a follow up. The priority here is to collect information and get a sense of overall trends. The survey becomes a failure if it is used as the primary way for people to share feedback with PostHog, something which we encourage them to do through more transparent channels. This is partly why we only do it every six months.

```
